### PR TITLE
OHRI-958 HIV Patient summary created in OHRI (short version)

### PR DESCRIPTION
**Pull request by kajambiya**

### CODE REVIEW

**Review by ODORA0 — APPROVED** [review]

File: packages/esm-commons-lib/src/components/encounter-tile/encounter-tile.component.tsx [hunk 1]
```diff
@@ -1,188 +1,154 @@
...
+        if (column.summaryConcept) {
+          if (column.isSummaryDaysCalculation) {
+            let summaryDate = getObsFromEncounter(value, column.summaryConcept);
+            if (summaryDate !== '--') {
+              const dateDifference = new Date().getTime() - new Date(summaryDate).getTime();
+              const totalDays = Math.floor(dateDifference / (1000 * 3600 * 24));
+              column.summaryObsValue = `${totalDays} days`;
+            }
+          } else {
+            column.summaryObsValue = getObsFromEncounter(value, column.summaryConcept, column.isConceptSummaryDate);
```
Comment (pirupius) [inline packages/esm-commons-lib/src/components/encounter-tile/encounter-tile.component.tsx:74]: This component needs to be reusable, that means it should be generic and not have specific logic. These need to be moved away from this component and added to the specific logic added to the specific implementations

File: packages/esm-commons-lib/src/components/encounter-tile/encounter-tile.component.tsx [hunk 2]
```diff
@@ -1,188 +1,154 @@
...
+      latestDate = artStopDate;
+      latestDateConcept = stopDate;
+    }
+    if (artRestartDate != null) {
+      latestDate = artRestartDate;
+      latestDateConcept = restartDate;
+    }
+
+    return latestDateConcept;
+  };
```
Comment (pirupius) [inline packages/esm-commons-lib/src/components/encounter-tile/encounter-tile.component.tsx:128]: same applies to these

**Review by pirupius — COMMENTED** [review]
@kajambiya you need to make a follow up PR to fix these. This PR shouldn't have been merged in it's current state because it's not reusable in other scenarios.

**Comment by pirupius** [comment]
@ODORA0 take note of the comments as well for the next time

**Pull request merged**